lambda/update/parse/changes.go

```diff
@@ -92,19 +101,29 @@ func Validate(fn func() []shared.FieldError) Option {
 // Then to parse to a string s do:
 //
 //	parser.Field("/thing", &s)
-func (p *Parser) Field(key string, v any, opts ...Option) *Parser {
+func (p *Parser) Field(key string, existing any, opts ...Option) *Parser {
 	options := fieldOpts{}
 	for _, opt := range opts {
 		options = opt(options)
 	}
 
 	for i, change := range p.changes {
 		if change.Key == key {
-			if !bytes.Equal(change.Old, []byte("null")) {
-				p.errors = append(p.errors, shared.FieldError{Source: change.Source("/old"), Detail: "must be null"})
+			// Adding for expand phase - will be removed once consumers are updated
+			if options.mustMatchExisting {
```

Comment:
or if you are keeping this, maybe have an `else` with the previous behaviour

Reply:
I'm not applying this option anywhere (other than in tests to prove it works) - it's more like a flag that I'll be removing once the consumer side has been updated. I can't leave the null check in as I need to update the consumer to send non-null values.

Reply:
:+1: